plugin/src/main/java/com/denizenscript/denizen/utilities/BukkitImplDeprecations.java

```diff
@@ -344,6 +344,15 @@ public class BukkitImplDeprecations {
     // Bad candidate for bumping, targets extremely commonly used naming, some of which may be hard to remove (eg stored in flag data).
     public static Warning oldSpigotNames = new FutureWarning("oldSpigotNames", "Several features (particles, entities, etc.) had alternative naming added by Spigot, which are now deprecated in favor of the official Minecraft naming; see relevant feature's meta docs for more information.");
 
+    // Added 2024/12/23, deprecate officially by 2028.
+    public static Warning timePeriod = new FutureWarning("timePeriod", "'time.period' is deprecated in favor of 'time_period'.");

```

Comment:
just do one deprecation for all of `world.time`, and use `pointlessSubtagPrefix`